bot.js

```diff
@@ -1327,6 +1370,52 @@ function command(message) {
         }
         log.info("Current server status:\n" + status);
         message.reply(status);
+    }
+    if (messageContent === config.commands.history) {
+        log.notice(
+            "Received song history command in text channel " +
+                message.channel.name +
+                ", server " +
+                message.guild.name +
+                "."
+        );
+        const url = config.API.aria.prefix + config.API.aria.history;
+
+        log.info("Making a request to " + url);
+        request.get({ url, form: {} }, (err, response, body) => {
+            if (!err && body != null) {
+                log.info("Request succeeded.");
+                log.debug("Received body:");
+                log.debug(body);
+                let history;
+                if ((body = "" || (history = JSON.parse(body)).length === 0)) {
+                    log.info("Results are empty. Replying appropriately.");
+                    message.reply("Cadence has no song history at the moment.");
```

Comment:
@kenellorando feel free to commend on this response and the others for `history`

Reply:
Looks fine. Cadence's UI just says `No history available (yet).`

https://github.com/kenellorando/cadence/blob/362800041145694dbf61da01833253078b670941/cadence/server/public/js/aria.js#L96-L97